src/components/ReportActionItem/ReportPreview.js

```diff
@@ -0,0 +1,159 @@
+
+    /** Active IOU Report for current report */
+    iouReport: PropTypes.shape({
+        /** Email address of the manager in this iou report */
+        managerEmail: PropTypes.string,
+
+        /** Email address of the creator of this iou report */
+        ownerEmail: PropTypes.string,
+
+        /** Outstanding amount in cents of this transaction */
+        total: PropTypes.number,
+
+        /** Currency of outstanding amount of this transaction */
+        currency: PropTypes.string,
+
+        /** Does the iouReport have an outstanding IOU? */
+        hasOutstandingIOU: PropTypes.bool,
+    }),
+
+    /** Session info for the currently logged in user. */
+    session: PropTypes.shape({
+        /** Currently logged in user email */
+        email: PropTypes.string,
+    }),
+
+    /** Popover context menu anchor, used for showing context menu */
+    contextMenuAnchor: PropTypes.shape({current: PropTypes.elementType}),
+
+    /** Callback invoked when View Details is pressed */
+    onViewDetailsPressed: PropTypes.func,
+
+    /** Callback for updating context menu active state, used for showing context menu */
+    checkIfContextMenuActive: PropTypes.func,
+
+    /** Whether the IOU is hovered so we can modify its style */
+    isHovered: PropTypes.bool,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    contextMenuAnchor: null,
+    isHovered: false,
+    iouReport: {},
+    onViewDetailsPressed: () => {},
+    checkIfContextMenuActive: () => {},
+    session: {
+        email: null,
+    },
+};
+
+const ReportPreview = (props) => {
+    const reportAmount = CurrencyUtils.convertToDisplayString(props.iouReport.total, props.iouReport.currency);
+    const managerEmail = props.iouReport.managerEmail || '';
+    const managerName = ReportUtils.getDisplayNameForParticipant(managerEmail, true);
+    const isCurrentUserManager = managerEmail === lodashGet(props.session, 'email', null);
+    return (
+        <View style={[styles.chatItemMessage, styles.mt4]}>
+            {_.map(props.action.message, (index) => (
+                <Pressable
+                    key={`ReportPreview-${props.action.reportActionID}-${index}`}
+                    onPress={props.onViewDetailsPressed}
+                    onPressIn={() => DeviceCapabilities.canUseTouchScreen() && ControlSelection.block()}
+                    onPressOut={() => ControlSelection.unblock()}
+                    onLongPress={(event) => showContextMenuForReport(event, props.contextMenuAnchor, props.chatReportID, props.action, props.checkIfContextMenuActive)}
+                    style={[styles.flexRow, styles.justifyContentBetween]}
+                    focusable
+                >
+                    <View>
+                        {props.iouReport.hasOutstandingIOU ? (
+                            <Text style={[styles.chatItemMessage, styles.cursorPointer]}>{props.translate('iou.payerOwesAmount', {owner: managerName, amount: reportAmount})}</Text>
```

Comment:
The key is `payer` not `owner` https://github.com/Expensify/App/blob/0b4f4e7163a61692bdae18c5b1cb4c40c816fb20/src/languages/en.js#L328

```suggestion
                            <Text style={[styles.chatItemMessage, styles.cursorPointer]}>{props.translate('iou.payerOwesAmount', {payer: managerName, amount: reportAmount})}</Text>
```

Though then this will look like:
<img width="1512" alt="image" src="https://github.com/Expensify/App/assets/4073354/68d8e15b-d207-44e7-9377-4557e7105e2c">

I (me the user viewing this) is `David` in this context... should it be `You owe $42`?